web/client/plugins/DrawerMenu.jsx

```diff
@@ -78,9 +79,13 @@ const DrawerMenu = React.createClass({
             menuOptions: {},
             singleSection: true,
             buttonClassName: "square-button",
-            disabled: false
+            disabled: false,
+            resetWidth: () => {}
         };
     },
+    componentWillUnmount() {
+        this.props.resetWidth();
```

Comment:
I don't think this is needed. If the control is not enabled, width is 0, we don't need an additional property